services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/SwitchOperationsBolt.java

```diff
@@ -56,7 +59,8 @@
 import java.util.Collections;
 import java.util.List;
 
-public class SwitchOperationsBolt extends PersistenceOperationsBolt {
+public class SwitchOperationsBolt extends PersistenceOperationsBolt implements ILinkOperationsServiceCarrier,
```

Comment:
ILinkOperationsServiceCarrier - this definitely doesn't correspond OpenKilda coding conventions: no special prefixes (like "I" here) or suffixes are used for naming.

Reply:
I made this interface aligned with existing ones in this topology. Should I rename them instead? https://github.com/telstra/open-kilda/tree/develop/services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/services